src/test/java/marquez/api/JobRunResponseTest.java

```diff
@@ -78,7 +78,7 @@ public void testHashCodeEquality() {
   }
 
   @Test
-  public void testJobRunInequalityOnUUID() {
+  public void testJobRunInequalityOnUuid() {
     JobRunResponse jr2 =
```

Comment:
This was a recommended change for my code in one of the other PRs, so I decided to also change it in the last couple places in the project where this was the case.

Reply:
Fixed in the reviews stacked atop this.